### ISSUE TITLE
(cli): re-bootstrapping wont update all tags

### ISSUE THREAD

**Issue by Wurstnase** [aws/aws-cdk#17406]
### What is the problem?

First bootstrap will create SSM parameter with tags. In our case we had publish the wrong tags and want to change the tags. Unfortunately the tags for the parameter are not updated.

### Reproduction Steps

npx cdk bootstrap --tags mytag=old_value
npx cdk bootstrap --tags mytag=new_value

### What did you expect to happen?

All tags are updated.

### What actually happened?

Nothing. Tags for the ssm parameter are unchanged.

### CDK CLI Version

1.131

### Node.js Version

14.x

### OS

Linux/MacOS

### Language

Python

**Comment by rix0rrr** [comment]
> What actually happened?
>
>Nothing. Tags for the ssm parameter are unchanged.

Can you be more specific?

Did nothing actually happen? As in, no tags got updated? Not those of an the S3 bucket, not those of the IAM roles?

Or did just the SSM parameter tags not get updated?

**Comment by Wurstnase** [comment]
I will check it again what happens.

What I've remember, only the tags for the SSM are not updated.

**Comment by Wurstnase** [comment]
> Or did just the SSM parameter tags not get updated?

Yes, SSM parameter are not updated. I just checked it with CDK 2.10 bootstrapping.
New tags are not added nor old tags are updated.

**Comment by comcalvi** [comment]
CDK applies those tags to the bootstrap stack. CFN should apply the tags to every resource in the stack, and it seems that it does not.
